less/index.less

```diff
@@ -50,10 +74,86 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
+}
+html{
+	font-size: @standard; // 62.5%
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size:1.2rem; // 75%
+	background: linear-gradient(to bottom, @color1 0%, @color2 100%);
+	background-attachment: fixed;
+}
+
+/* Container Styles */
+
+
+.container {
+	width:60%;
+	margin: auto;
+	padding: 2%;
+	a {
+		color:white;
+	}
+	.navbar {
+		display: flex;
+		justify-content: flex-end;
+		background-color: @color2;
+
+		a {
+			vertical-align: center;
+			text-decoration: none;
+			font-size:1rem;
+			margin-right: 2%;
+			color: @linkColor; // Nav Anchor Color
+		}
+	}
+
+	h1, h2, h3 {
+		font-family:@bilbo-font;
+		font-size:1.6rem; // 1.6 * 62.5% = 100%
+	}
+
+	div {
+		padding:2%;
+	}
+
+	p {
+		max-width: 65ch;
+		font-size:1rem;
+		margin-left:1%;
+		text-indent: 1vw; //Indent our paragraphs 1% of the screenwidth.
+		padding:2%;
+	}
+
+	.intro{
+		padding: 2%;
+		background-color: lightyellow;
+		.header-portrait {
+			height: 30vh;
+		}
+	}
+	.skills ul {
+		list-style: disc inside none;
+	}
+	
+	table {
+		border-collapse: collapse;
+		border-spacing: 0;
+		.border();
+		.table-header {
+			background-color: @linkColor;
+		}
```

Comment:
I'd recommend adding something a little like
```
td {
    padding-left: 2%;
}
```
Maybe not just that one property, but something to space out each column in the table so they have a little more space left-to-right.